README.md

```diff
@@ -1 +1,69 @@
-# .github
\ No newline at end of file
+# CSRF (This is a community driven project)
+
+CSRF for hertz.
```

Comment:
What is CSRF and what does it used for ? - The least information you should mention here

Reply:
> What is CSRF and what does it used for ? - The least information you should mention here

done